rain_api_core/urs_util.py

```diff
@@ -254,21 +256,21 @@ def get_urs_creds():
 
 def user_profile_2_jwt_payload(user_id, access_token, user_profile):
     return {
-            # Do we want more items in here?
-            'first_name': user_profile['first_name'],
-            'last_name': user_profile['last_name'],
-            'email': user_profile['email_address'],
-            'urs-user-id': user_id,
-            'urs-access-token': access_token,
-            'urs-groups': user_profile['user_groups'],
-            'iat': int(time()),
-            'exp': get_exp_time(),
-        }
+        # Do we want more items in here?
```

Comment:
Should probably mark this with TODO

Reply:
Or just delete it

Reply:
I'd opt for delete

Reply:
Pretty safe to assume nobody wants more items in there, 3 years later.